### ISSUE TITLE
Enhancement - Garbage Collector Settings

### ISSUE THREAD

**Issue by BloodBlight** [Linux74656/SpaceEngineersLinuxPatches#56]
Adjusting the garbage collector seems to help with stability:

```
<configuration>        
    <runtime>
        <gcConcurrent enabled="false"/>
        <gcServer enabled="true" />
    </runtime>
<startup>
```

**Comment by arch-user-france1** [comment]
this can be done with admin tools and isn't required

**Comment by arch-user-france1** [comment]
Maybe it removes items but the user doesn't want it. I can have thousands of auto-generated ships (mod) and it's still stable. There's only like 1 message / 5 minutes that there are too many items

**Comment by BloodBlight** [comment]
> Maybe it removes items but the user doesn't want it. I can have thousands of auto-generated ships (mod) and it's still stable. There's only like 1 message / 5 minutes that there are too many items

I don't THINK this has anything to do with the in game object garbage collection.  This is for things like the .Net memory garbage collection and should NOT have any impact on gameplay.

But I could be wrong.

**Comment by arch-user-france1** [comment]
Ah I thaught the collector from the Game was meant, my mistake

Uhm yeah I agree with the message above. It shouldn't have any impact with Game experience - because wine / proton doesn't have problems with RAM. Mayve he just has 1MB RAM. I had 8GB on Windows and it was fine - now I've 40GB and I'm using Manjaro (well, Ubuntu seems ti be better but.. Manjaro has zero effect on game play - it's also slower write speed and freezes sometimes. Maybe I see a reason using it but I also hate ubuntu  

Well we should wait until he replies - or close if he doesn't

**Comment by BloodBlight** [comment]
Ya, I could be an edge case.  I was already having play freezing and this reduced it and decreased crashing.  But I think that might be my GPU.  I only have 4GBs of vRAM on an old RX 580.  How much do you have?

I have recorded some weird stuff that others are not seeing so...  Like blocks popping in and out of render based on were you stand.  I am good at breaking things...

**Comment by arch-user-france1** [comment]
Well I'm also good at breaking stuff

For example remote controllers - I broke them
Or ships with no power but they are flying (turning) around randomly uncontrollable ??!!! :-)

I have 40GB of RAM - a weak Intel I5 uhm idk 3GHz - boosts to 3.5GHz 4cores no multithreading. I'll change to AMD once I buy a new processor..

And NVidia GeForce GTX 1660 super (very good but cheap, all use cases, graphics on high or ultra is just normal for me with good fps (well the CPU makes lag spices sometimes), 6GB VRAM, 120FPS  (max) in Space Engineers (Windows, wine only 30-50), 600FPS Minecraft (Windows render distance 64, Optifine), 2000 FPS Ubuntu when standing still (because minecraft has problems with weak CPUs it's too laggy when walking with max render distance, render distance 12) uhm yeah just ask me something if you want)

**Comment by arch-user-france1** [comment]
> Ya, I could be an edge case. I was already having play freezing and this reduced it and decreased crashing. But I think that might be my GPU. I only have 4GBs of vRAM on an old RX 580. How much do you have?
> 
> I have recorded some weird stuff that others are not seeing so... Like blocks popping in and out of render based on were you stand. I am good at breaking things...

Well did you ever use Windows or just Linux?

**Comment by BloodBlight** [comment]
Yep, but not much.  It runs fine on Windows.  Don't know about the other stuff, didn't use it enough as I remembered how much I hated Windows when it rebooted on me for the third time.

The rest of my system if fairly modern, just Team Red.
